docs/UserGuide.adoc

```diff
@@ -72,6 +72,12 @@ Format:
 
 Format: `contacts find KEYWORD [MORE_KEYWORDS]`
 
+==== Select a contact: `contacts select`
```

Comment:
Will be removed on my PR, abit inconsistent but it will be replaced with `contacts assigned`

`tasks select` will be used more for populating details pane

Reply:
Noted.